test/providers/provider_codebuild.test.js

```diff
@@ -0,0 +1,96 @@
+const td = require('testdouble')
+const { describe } = require('yargs')
```

Comment:
This is the wrong kind of describe block.

Reply:
This was likely inserted automatically by your IDE. Test fixtures are typically injected into the environment as globals.

Reply:
```suggestion
```